index.html

```diff
@@ -250,7 +249,44 @@ <h3 class="title">Project Tile goes here</h3>
                 </div>
             </div>
         </section>
-    </div>
+
+        <footer class="information-footer">
+
+            <div class="contacts-foolter">
+                <div class="logo-footer">
+                    <a href="index.html"><img src="images/logo 1 (2).svg" alt="logo"></a>
+                </div>
+
+                <div class="content-foolter">
+                    <p class="text-footer">+55 83 988970601</p>
+                    <p class="text-footer">mikael.mesg@gmail.com</p>
+                    <a class="icon-footer" href=""><img src="images/Vector (2).svg" alt="logo-gthub"></a>
+                    <a class="icon-footer" href=""><img src="images/ant-design_twitter-circle-filled (1).svg" alt="logo-twitter"></a>
+                    <a class="icon-footer" href=""><img src="images/entypo-social_linkedin-with-circle (1).svg" alt="logo-linkedin"></a>
+                </div>
+            </div>
+
+            <div class="division">
```

Comment:
Acho que faz mais sentido chamar essa classe de `separator`
```suggestion
            <div class="separator">
```

Reply:
@mikaelelias

Reply:
feito!